public/docs/ts/_cache/tutorial/toh-pt5.jade

```diff
@@ -148,172 +168,220 @@ code-example(language="bash").
++makeExcerpt('app/app.component.1.ts', 'template-v2')
+
+block routerLink
   :marked
-    For more dynamic router links, learn about the *link parameters array* in the [Routing](../guide/router.html#link-parameters-array) chapter.
+    Notice the `routerLink` binding in the anchor tag.
+    We bind the `RouterLink` directive (another of the `RouterModule` directives) to a string
+    that tells the router where to navigate when the user clicks the link.
+
+    Since our link is not dynamic, we define a *routing instruction* with a **one-time binding** to our route **path**.
+    Looking back at the route configuration, we confirm that `'/heroes'` is the path of the route to the `HeroesComponent`.
+  .l-sub-section
+    :marked
+      Learn more about dynamic router links and the *link parameters array*
+      in the [Routing](../guide/router.html#link-parameters-array) chapter.
+
 :marked
-  Refresh the browser.  We see only the app title. We don't see the heroes list.
+  Refresh the browser.  We see only the app title and heroes link. We don't see the heroes list.
+
 .l-sub-section
   :marked
     The browser's address bar shows `/`.
     The route path to `HeroesComponent` is `/heroes`, not `/`.
     We don't have a route that matches the path `/`, so there is nothing to show.
     That's something we'll want to fix.
+
 :marked
-  We click the "Heroes" navigation link, the browser bar updates to `/heroes`,
+  We click the *Heroes* navigation link, the browser bar updates to `/heroes`,
   and now we see the list of heroes. We are navigating at last!
 
   At this stage, our `AppComponent` looks like this.
-+makeExample('toh-5/ts/app/app.component.2.ts',null, 'app/app.component.ts (v2)')
+
++makeExample('app/app.component.1.ts', 'v2', 'app/app.component.ts (v2)')
+
 :marked
   The  *AppComponent* is now attached to a router and displaying routed views.
   For this reason and to distinguish it from other kinds of components,
   we call this type of component a *Router Component*.
 
-
 :marked
   ## Add a *Dashboard*
+
   Routing only makes sense when we have multiple views. We need another view.
 
   Create a placeholder `DashboardComponent` that gives us something to navigate to and from.
-+makeExample('toh-5/ts/app/dashboard.component.1.ts',null, 'app/dashboard.component.ts (v1)')(format=".")
+
++makeExcerpt('app/dashboard.component.1.ts (v1)', '')
+
 :marked
   We’ll come back and make it more useful later.
 
   ### Configure the dashboard route
-  Go back to `app.routing.ts` and teach it to navigate to the dashboard.
 
-  Import the `DashboardComponent` so we can reference it in the dashboard route definition.
+  Go back to `!{_appRoutingTsVsAppComp}` and teach it to navigate to the dashboard.
 
-  Add the following `'Dashboard'` route definition to the `Routes` array of definitions.
-+makeExample('toh-5/ts/app/app.routing.2.ts','dashboard-route', 'app/app.routing.ts (Dashboard route)')(format=".")
+  Import `DashboardComponent` and 
```

Comment:
Perhaps it'd be better to show the dashboard component import, as well? That would mean changing the intro text to something like:

Import the dashboard component and add a route definition to the ... definitions: